nnvm/src/pass/gradient.cc

```diff
@@ -143,23 +143,23 @@ Graph Gradient(Graph src) {
         << "because it is unreachable from the outputs.";
   }
 
-  // construct mirror reduece memory strategy if needed
+  // construct mirror as memory reduction strategy if needed
```

Comment:
is this mirror functionality used somewhere? at least I couldn't find usages in MXNet